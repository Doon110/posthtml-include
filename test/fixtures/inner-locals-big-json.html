<h1>index</h1>

<include src="./test/fixtures/includes/5.html">
  {
    "_id": "5f80a36f5be511b1fc7afb87",
    "index": 0,
    "guid": "0bc2e2bc-69f5-4fa4-9d05-3167ed5e33f2",
    "isActive": false,
    "balance": "$1,307.93",
    "picture": "http://placehold.it/32x32",
    "age": 28,
    "eyeColor": "blue",
    "name": "Blankenship Schneider",
    "gender": "male",
    "company": "EXOVENT",
    "email": "blankenshipschneider@exovent.com",
    "phone": "+1 (885) 459-3832",
    "address": "420 Farragut Place, Brazos, Kentucky, 2848",
    "about": "Ex sit proident velit ut enim dolore. Tempor proident aliqua fugiat tempor excepteur magna anim elit.",
    "registered": "2014-06-12T05:37:51 +03:00",
    "latitude": -71.554141,
    "longitude": -73.693833,
    "greeting": "Hello, Blankenship Schneider! You have 4 unread messages.",
    "favoriteFruit": "banana",   
    "tags": [
      "esse",
      "culpa",
      "labore",
      "sint",
      "ut",
      "eiusmod",
      "consectetur"
    ],
    "friends": [
      {
        "id": 0,
        "name": "Raymond Salinas"
      },
      {
        "id": 1,
        "name": "Woodward Sampson"
      },
      {
        "id": 2,
        "name": "Christian Long"
      }
    ]
  }
</include>