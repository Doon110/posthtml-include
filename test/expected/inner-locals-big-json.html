<h1>index</h1>

<ul>
  <li>5f80a36f5be511b1fc7afb87</li>
  <li>0</li>
  <li>0bc2e2bc-69f5-4fa4-9d05-3167ed5e33f2</li>
  <li>false</li>
  <li>$1,307.93</li>
  <li>http://placehold.it/32x32</li>
  <li>28</li>
  <li>blue</li>
  <li>Blankenship Schneider</li>
  <li>male</li>
  <li>EXOVENT</li>
  <li>blankenshipschneider@exovent.com</li>
  <li>+1 (885) 459-3832</li>
  <li>420 Farragut Place, Brazos, Kentucky, 2848</li>
  <li>Ex sit proident velit ut enim dolore. Tempor proident aliqua fugiat tempor excepteur magna anim elit.</li>
  <li>2014-06-12T05:37:51 +03:00</li>
  <li>-71.554141</li>
  <li>-73.693833</li>
</ul>